schema/library.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "library_item_id": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 67
```

Comment:
I think that we might not want a maximum for now. @leijerry888, what do you think about it?

Reply:
> I think that we might not want a maximum for now. @leijerry888, what do you think about it?

That's correct. We do not want a maximum.

Reply:
Updated